biggus/tests/unit/test_Array.py

```diff
@@ -240,6 +235,20 @@ def test_other_no_good(self):
             a // None
 
 
+class Test___div__(unittest.TestCase, AssertElementwiseMixin):
+    def test_other_array(self):
+        a = FakeArray([2, 4])
+        b = FakeArray([2, 4])
+        r = operator.div(a, b)
```

Comment:
There is no special meaning for __div__ in py3k. I therefore suggest not trying to get an operator to implement these tests (both of them), and instead, just call the __div__ methods directly. (e.g. ```a.__div__(b)```)